src/core/searchResult/Specifications.ts

```diff
@@ -1,6 +1,7 @@
-const { SearchResultBase } = require('./SearchResultBase');
+import { SearchResultBase } from './SearchResultBase';
+import { JSONObject } from '../../utils/interfaces';
 
-class SpecificationsSearchResult extends SearchResultBase {
+export class SpecificationsSearchResult extends SearchResultBase<JSONObject> {
```

Comment:
I know I should have created a proper `Specification` interface but I'm lazy to do it since I doubt someone is using this and we need a refactor of this module.
The `JSONObject` type works like a POJO